### ISSUE TITLE
No packages with master branch

### ISSUE THREAD

**Issue by badfiles** [webmin/webmin#400]
New packages [here](http://download.webmin.com/devel/) do not include latest master changes.

**Comment by jcameron** [comment]
Correct, they aren't continually rebuilt to track the master branch in git.

**Comment by badfiles** [comment]
Is it on purpose or by mistake? Until v. 1.801 github's master branch and dev packages were very close.
The packages' build dates update every day, but the contents stay the same.

**Comment by swelljoe** [comment]
There has never been an actual daily build of Webmin; sometimes more dev builds get pushed out than usual, when Jamie's dealing with a particular issue that needs some back-and-forth with a bug reporter or feature requester to get it sorted out fully, but there isn't currently daily build automation in place. But, the devel builds are pushed out by Jamie when he triggers it, not every day.

I'm experimenting with CI to potentially add automated daily builds for Webmin 2.0 development, which we're branching in a few weeks; we're grinding out a bunch of updates to get ready for that.

If you're developing on Webmin (great!) and want to track the latest git version on your development environment, you'll probably want to use a git checkout directly, instead of from packages. If you just need one specific fix that's gone into the master branch, you might grab just the effected module or files.

Anyway, daily builds are coming eventually. We have a lot of stuff to do to get it integrated into a CI system, with automated tests and static analysis and such. That work is ongoing for the 2.0 branch.

**Comment by badfiles** [comment]
I see, thank you.

**Comment by iliaross** [comment]
All builds now run on every repository change, which typically happens more than once a day.

https://download.webmin.dev
